mappings/net/minecraft/test/TimedTaskRunner.mapping

```diff
@@ -2,9 +2,24 @@ CLASS net/minecraft/class_4693 net/minecraft/test/TimedTaskRunner
 	FIELD field_21457 test Lnet/minecraft/class_4517;
 	FIELD field_21458 tasks Ljava/util/List;
 	FIELD field_21459 tick J
+	METHOD <init> (Lnet/minecraft/class_4517;)V
+		ARG 1 gameTest
 	METHOD method_23643 runSilently (J)V
 		ARG 1 tick
 	METHOD method_23644 runReported (J)V
 		ARG 1 tick
 	METHOD method_23645 runTasks (J)V
 		ARG 1 tick
+	METHOD method_36077 (ILjava/lang/Runnable;)Lnet/minecraft/class_4693;
+		ARG 2 task
+	METHOD method_36078 createAndAdd (JLjava/lang/Runnable;)Lnet/minecraft/class_4693;
+		ARG 1 duration
+		ARG 3 task
+	METHOD method_36079 createAndAddDurationless (Ljava/lang/Runnable;)Lnet/minecraft/class_4693;
```

Comment:
Does this need `Durationless`? It looks like the duration variant just takes it as an extra parameter so they shouldn't conflict.